js/components/ViewPagerIndicator.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react';
```

Comment:
I'm going to be a bit picky, could you add some documentation explaining what this component does?